### PR TITLE
Improve tf.batch_to_space documentation. Refer issue #34005

### PR DESCRIPTION
**Pull request by nikochiko**
Made minor changes to make code more suitable for GCI challenge. Refer #34005 
The changes have been made in order to make the code suitable for a Google Code-In Documentation and Research challenge.
The task now is to clean the documentation so that it is more clear. There are too many code blocks in-between the text.
Some examples are placed in the args section for illustration. A better way to handle that would be to add those examples in a separate "Examples" section (consistent with other docs) and add pointers in the "Args" section.
Similarly, some other code elements could be eliminated from the text by adding pointers.
Clarify the crops part in the "Args" section further.

### CODE REVIEW

**Comment by nikochiko** [comment]
@mihaimaruseac Sorry, didn't notice the r2.x branch.

File: tensorflow/python/ops/array_ops.py [hunk 1]
```diff
@@ -3571,31 +3571,35 @@ def batch_to_space_v2(input, block_shape, crops, name=None):  # pylint: disable=
...
-          `block_shape = [2, 2]`, and `crops = [[0, 0], [0, 0]]`:  ``` x =
-            [[[[1], [3]], [[9], [11]]], [[[2], [4]], [[10], [12]]], [[[5], [7]],
-            [[13], [15]]], [[[6], [8]], [[14], [16]]]] ```
-      The output tensor has shape `[1, 4, 4, 1]` and value:  ``` x = [[[1], [2],
-        [3],  [4]], [[5],   [6],  [7],  [8]], [[9],  [10], [11],  [12]], [[13],
-        [14], [15],  [16]]] ```  (4) For the following input of shape `[8, 1, 3,
-        1]`,
+      Some Examples:
+      (1) For the following input of shape `[4, 1, 1, 1]`,
+         `block_shape = [2, 2]`, and `crops = [[0, 0], [0, 0]]`:  ``` [[[[1]]],
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/array_ops.py]: I think the triple backquotes should be on their own line, here and below

**Review by mihaimaruseac — COMMENTED** [review]
Looks good to me, minor nit regarding documentation

File: tensorflow/python/ops/array_ops.py (hunk 1 above)
Comment (nikochiko) [reply tensorflow/python/ops/array_ops.py]: I did it lightly so as to make it a GCI task. But anyway, this is almost completed.

**Comment by nikochiko** [comment]
@mihaimaruseac I have a bad internet connection here. Sorry, didn't mean to re-request review after you assigned another reviewer.

**Comment by rthadur** [comment]
@nikochiko thank you , can you please check the sanity errors [here](https://source.cloud.google.com/results/invocations/5c77a330-068d-403f-9d59-2264e293e670/log) and fix them.

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**